docs/src/main/sphinx/routines/introduction.md

```diff
@@ -66,6 +66,8 @@ In this scenario, the following commands can be used:
 * [](/sql/create-function) to create and store a routine.
 * [](/sql/drop-function) to remove a routine.
 * [](/sql/show-functions) to display a list of routines in a catalog.
+* [](/sql/show-create-function) to display the SQL statement that created the
```

Comment:
reword / expand to explain that this essentially shows the source code of the routine and therefore kinda also can be used to explain what it does in more detail than the description available with show functions.

It might be worth doing that here or in the linked doc .. or both maybe